crates/aptos-push-metrics/src/lib.rs

```diff
@@ -185,13 +188,13 @@ impl MetricsPusher {
                 .to_string()
         });
         let push_metrics_labels = Self::push_metrics_extra_labels(chain_name, &namespace);
-        let worker_thread =
-            Self::start_worker_thread(rx, push_metrics_labels, chain_name, &namespace);
+        let pusher = Self::start(push_metrics_labels);
 
-        Self {
-            worker_thread,
-            quit_sender: tx,
-        }
+        info!(
+            "Execution dashboard link: {} ",
+            Self::get_dashboard_link(chain_name, &namespace)
+        );
+        pusher
```

Comment:
why not move these to the benchmark itself?

Reply:
We can but I think this is okay as well :)